src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -137,87 +137,93 @@ public override MetadataFile FindMetadataFile(Series series, string path)
 
         public override MetadataFileResult SeriesMetadata(Series series)
         {
-            if (!Settings.SeriesMetadata)
+            var xmlResult = string.Empty;
+            if (Settings.SeriesMetadata)
```

Comment:
If "Series Metadata" is unchecked and "Series Metadata URL" is checked then only the TVDB URL will be written to tvshow.nfo. This creates a [Parsing NFO](https://kodi.wiki/view/NFO_files/Parsing), which supplies Kodi with the URL to be able to scrape the info from TheTVDB itself.

If both checkboxes are checked then the usual metadata XML plus the TVDB URL will be written to tvshow.nfo. This creates a [Combination NFO](https://kodi.wiki/view/NFO_files/Combination), which allows Kodi to parse the metadata XML as usual but also provides the TVDB URL so it can scrape additional info as necessary.

The only time a tvshow.nfo file would not be written would be when both checkboxes are unchecked.

Reply:
I think this needs to be clarified, with the addition of the URL it's not clear that if the option that creates tvshow.nfo is unchecked the file will still be created.

`Settings.SeriesMetadata` would be `tvshow.nfo with full series metadata`
`Settings.SeriesMetadataUrl` would be `tvshow.nfo with TheTVDB show URL (can be combined with 'Series Metadata')`